tools/cli.js

```diff
@@ -116,9 +124,41 @@ commandProcessor.complete = (inputs, depth) => {
 
 commandProcessor.call = (interfaceName, methodName, args, callback) => {
   if (!state.client) return callback(new Error('Not connected'));
+
+  interfaceName = tryComplete(interfaceName, Object.keys(state.api));
```

Comment:
@aqrln actually that may really be the case, but I'd like to at least add cli option like
`-ac` / `--advancedCompletion` that'll enable this behavior as this may be convenient for advanced users. But I guess we will better make it in different PR and for now either add hard-coded config variable or comment it out comepletely untill we add the option? @belochub?